### ISSUE TITLE
Emby requires transcoding while Plex does not

### ISSUE THREAD

**Issue by HazCod** [MediaBrowser/Emby#1188]
Test file: [big buck bunny h264 mov](http://download.blender.org/peach/bigbuckbunny_movies/big_buck_bunny_1080p_h264.mov)

Steps to reproduce:
Plex: Add to library, try to play. No transcoding. (I checked CPU consumption on my laptop)
Emby: Add to library, create user that is not allowed to transcode (video/audio). 'No compatible stream detected'

Both tested in chrome.

File info:
```
niels@django:~$ mediainfo /tmp/Movies/Big\ Buck\ Bunny/big_buck_bunny_1080p_h264.mov 
General
Complete name                            : /tmp/Movies/Big Buck Bunny/big_buck_bunny_1080p_h264.mov
Format                                   : MPEG-4
Format profile                           : QuickTime
Codec ID                                 : qt  
File size                                : 692 MiB
Duration                                 : 9mn 56s
Overall bit rate                         : 9 725 Kbps
Encoded date                             : UTC 2008-05-27 18:40:35
Tagged date                              : UTC 2008-05-27 18:43:05
Writing library                          : Apple QuickTime 7.4.1
com.apple.quicktime.player.movie.audio.m : (Binary)

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Main@L4.1
Format settings, CABAC                   : No
Format settings, ReFrames                : 2 frames
Format settings, GOP                     : M=2, N=24
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 9mn 56s
Bit rate                                 : 9 283 Kbps
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 24.000 fps
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.187
Stream size                              : 660 MiB (95%)
Language                                 : English
Encoded date                             : UTC 2008-04-21 20:24:31
Tagged date                              : UTC 2008-05-27 18:43:05
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709

Audio
ID                                       : 3
Format                                   : AAC
Format/Info                              : Advanced Audio Codec
Format profile                           : LC
Codec ID                                 : 40
Duration                                 : 9mn 56s
Source duration                          : 9mn 56s
Bit rate mode                            : Constant
Bit rate                                 : 448 Kbps
Channel(s)                               : 6 channels
Channel positions                        : Front: L C R, Side: L R, LFE
Sampling rate                            : 48.0 KHz
Compression mode                         : Lossy
Stream size                              : 31.1 MiB (4%)
Source stream size                       : 31.1 MiB (4%)
Language                                 : English
Encoded date                             : UTC 2008-05-27 18:40:12
Tagged date                              : UTC 2008-05-27 18:43:05

Other
ID                                       : 2
Type                                     : Time code
Format                                   : QuickTime TC
Duration                                 : 9mn 56s
Time code of first frame                 : 00:00:00:00
Time code settings                       : Striped
Language                                 : English
Encoded date                             : UTC 2008-04-21 20:24:31
Tagged date                              : UTC 2008-05-27 18:43:05
```

**Comment by HazCod** [comment]
Same result when converted to mp4;
```
General
Complete name                            : /tmp/Movies/Big Buck Bunny/big_buck_bunny_1080p_h264.mp4
Format                                   : MPEG-4
Format profile                           : Base Media
Codec ID                                 : isom
File size                                : 724 MiB
Duration                                 : 9mn 56s
Overall bit rate                         : 10.2 Mbps
Writing application                      : Lavf56.40.101

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : Main@L4.1
Format settings, CABAC                   : No
Format settings, ReFrames                : 2 frames
Format settings, GOP                     : M=2, N=24
Codec ID                                 : avc1
Codec ID/Info                            : Advanced Video Coding
Duration                                 : 9mn 56s
Bit rate                                 : 9 283 Kbps
Width                                    : 1 920 pixels
Height                                   : 1 080 pixels
Display aspect ratio                     : 16:9
Frame rate mode                          : Constant
Frame rate                               : 24.000 fps
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.187
Stream size                              : 660 MiB (91%)
Language                                 : English
Color primaries                          : BT.709
Transfer characteristics                 : BT.709
Matrix coefficients                      : BT.709

Audio #1
ID                                       : 2
Format                                   : AAC
Format/Info                              : Advanced Audio Codec
Format profile                           : LC
Codec ID                                 : 40
Duration                                 : 9mn 56s
Bit rate mode                            : Constant
Bit rate                                 : 256 Kbps
Channel(s)                               : 2 channels
Channel positions                        : Front: L R
Sampling rate                            : 48.0 KHz
Compression mode                         : Lossy
Stream size                              : 18.2 MiB (3%)
Language                                 : English

Audio #2
ID                                       : 3
Format                                   : AC-3
Format/Info                              : Audio Coding 3
Mode extension                           : CM (complete main)
Format settings, Endianness              : Big
Codec ID                                 : ac-3
Duration                                 : 9mn 56s
Bit rate mode                            : Constant
Bit rate                                 : 640 Kbps
Channel(s)                               : 6 channels
Channel positions                        : Front: L C R, Side: L R, LFE
Sampling rate                            : 48.0 KHz
Bit depth                                : 16 bits
Compression mode                         : Lossy
Stream size                              : 45.5 MiB (6%)
Language                                 : English
```

**Comment by psxlover** [comment]
From my experience any file you play through browser will be transcoded.

**Comment by HazCod** [comment]
Really? That is just ridiculous.

**Comment by nyanloutre** [comment]
I also have an error when I try to play a mkv with h264 and aac audio with Chromium if the transcode is disable.
I have Emby server v3.0.5724.6 and Chromium 45.0.2454.85 on Ubuntu.

**Comment by cowai** [comment]
@LukePulverenti @nyanloutre
I am on Google Chrome (not chromium) on Ubuntu with v3.0.5724.6 and no mkv file is remuxed/streamcopied. Everything is reencoded.
I can confirm that the mkv files that I tried is standard h264 files. If I remux them manually with ffmpeg .. -c copy .. Chrome plays them perfectly without the server reencoding.

**Comment by cowai** [comment]
Also, when I play mp4 files in chromium it reencodes to libx264 on the server.

**Comment by cowai** [comment]
@LukePulverenti Can you help me explain if this is caused by bug in the frontend or backend?
Also is there some way of forcing to streamcopy?

**Comment by cowai** [comment]
@LukePulverenti I tried the latest version now (3.0.5781.0). And the problem persists.

To reproduce this bug, use a linux on client and server. ( I have Ubuntu 14.04 on  both).
Use Google Chrome (not chromium). Try to play any mkv in the web client. It will always transcode no matter what. I have tried with a lots of different shows and it transcodes on all of them.
If I do ffmpeg -i show.mkv -c copy show.mp4, it plays without transcoding.
